omod/src/main/webapp/resources/scripts/visit/encounterTypeConfig.js

```diff
@@ -305,8 +280,40 @@ angular.module("encounterTypeConfig", ["constants"])
             editUrl: "/htmlformentryui/htmlform/editHtmlFormWithStandardUi.page?patientId={{visit.patient.uuid}}&visitId={{visit.uuid}}&encounterId={{encounter.uuid}}&definitionUiResource=pihcore:htmlforms/section-delivery.xml&returnUrl={{returnUrl}}&breadcrumbOverride={{breadcrumbOverride}}"
         }
 
-        /* Define Encounter Types */
-        var encounterTypeConfig = {
+        /**
+         * Define Encounter Types
+         * Should support all of the following formats:
+         *
+         * encounterTypes['some-uuid'] = {
+         *    // single config
+         * }
+         *
+         * encounterTypes['some-uuid'] = {
+         *    DEFAULT: {
+         *      // default config
+         *    }
+         *    SPECIFIC_COUNTRY: {
+         *      // specific country config
+         *    }
+         * }
+         *
+         *
+         * encounterTypes['some-uuid'] = {
+         *    DEFAULT: {
+         *      // default config
+         *    }
+         *    SPECIFIC_COUNTRY: {
+         *      DEFAULT: {
+         *        // specific country config
+         *      }
+         *      SPECIFIC_SITE: {
+         *        // specific site config
+         *      }
+         *    }
+         * }
+         **/
+
```

Comment:
I had hoped to support having a country or site "automatically" pick up the forms from the proper site or country sub-directory, but I realized this isn't (easily) possible because all this config is happening on the client side so we'd have to query the server to see if a country or site-specific form was available and otherwise fall back to the default.

So configuring country and site-specific forms will still require editing the encounterTypeConfig as shown above... however, we probably want to support completely different configs per country anyway, and with this change, we can do that without defining a new encounter type.